src/chicago_participatory_urbanism/geocoder_api.py

```diff
@@ -24,7 +19,7 @@ class GeoCoderAPI:
     'headers to query socrata api'
     api_header = {
         'Accept': 'application/json',
-        'X-App-Token': os.getenv('app_token'),
+        'X-App-Token': os.environ['app_token'],
```

Comment:
raises a loud error if the env variable isn't provided, as opposed to `.getenv` which just returns a `None` object

Reply:
sounds good!